autogpt/core/ability/builtins/file_operations.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import os
+
+from autogpt.core.ability.base import Ability, AbilityConfiguration
+from autogpt.core.ability.schema import AbilityResult, ContentType, Knowledge
+from autogpt.core.workspace import Workspace
+
+
+class ReadFile(Ability):
+    default_configuration = AbilityConfiguration(
```

Comment:
This line causes error when trying to employ ReadFile class, because AbilityConfiguration does not provide any 'location' which is a required field.
![Screenshot from 2023-07-31 11-50-56](https://github.com/Significant-Gravitas/Auto-GPT/assets/39694513/36ced946-8ae8-4d37-aeb0-c02f030781a3)